kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx

```diff
@@ -58,20 +51,19 @@ const ClustersWidget: React.FC<Props> = ({
             checked={showOfflineOnly}
             onChange={handleSwitch}
           />
-          <label htmlFor="switchRoundedDefault">
-          </label>
+          <label htmlFor="switchRoundedDefault" />
         </Indicator>
       </MetricsWrapper>
 
-      {clusterList.map((chunk, idx) => (
-        <div className="columns" key={`dashboard-cluster-list-row-key-${idx}`}>
-          {chunk.map((cluster, idx) => (
-            <ClusterWidget {...cluster} key={`dashboard-cluster-list-item-key-${idx}`}/>
+      {clusterList.map((chunkItem) => (
+        <div className="columns" key={v4()}>
```

Comment:
why is it better? I believe v4 assigns new key for each `<div>` every time, so it is useless. Moreover, it can cause uncontrolled re-renderings of elements

Reply:
Fixed it, replaced key value with unique fields. Unfortunately, not all of the data had it, so I added it to them.